src/main/java/io/cdap/plugin/cloud/vision/transform/ExtractorTransformConfig.java

```diff
@@ -48,7 +48,6 @@
   @Name(ExtractorTransformConstants.OUTPUT_FIELD)
   @Description("Field to store the extracted image features. If the specified output field name already exists " +
     "in the input record, it will be overwritten.")
-  @Macro
```

Comment:
why can't this be a macro? Generally everything should be a macro unless there is a specific reason.

Reply:
fixed